diff --git a/charmap.txt b/charmap.txt
index 477448c9c5..8f307f672e 100644
--- a/charmap.txt
+++ b/charmap.txt
@@ -155,6 +155,36 @@ SUPER_RE    = A0
 TALL_PLUS   = FC 0C FB
 '$'         = FF
 
+@ Hebrew
+'א'         = D5
+'ב'         = D6
+'ג'         = D7
+'ד'         = D8
+'ה'         = D9
+'ו'         = DA
+'ז'         = DB
+'ח'         = DC
+'ט'         = DD
+'י'         = DE
+'כ'         = DF
+'ל'         = E0
+'מ'         = E1
+'נ'         = E2
+'ס'         = E3
+'ע'         = E4
+'פ'         = E5
+'צ'         = E6
+'ק'         = E7
+'ר'         = E8
+'ש'         = E9
+'ת'         = EA
+'ך'         = EB
+'ם'         = EC
+'ן'         = ED
+'ף'         = EE
+
+'ץ'         = F1
+
 @ Hiragana
 'あ' = 01
 'い' = 02
diff --git a/graphics/fonts/latin_normal.png b/graphics/fonts/latin_normal.png
index ade914858f..698c76fef9 100644
Binary files a/graphics/fonts/latin_normal.png and b/graphics/fonts/latin_normal.png differ
diff --git a/src/fonts.c b/src/fonts.c
index 8ef32bfef2..a68bed1eef 100644
--- a/src/fonts.c
+++ b/src/fonts.c
@@ -159,8 +159,8 @@ ALIGNED(4) const u8 gFontNormalLatinGlyphWidths[] = {
     8,  6,  6,  6,  6,  6,  6,  6,  6,  6,  6,  4,  6,  3,  6,  3,
     6,  6,  6,  3,  3,  6,  6,  6,  3,  7,  6,  6,  6,  6,  6,  6,
     6,  6,  6,  6,  6,  6,  6,  6,  6,  6,  6,  6,  6,  6,  6,  6,
-    6,  6,  6,  6,  6,  6,  6,  6,  6,  6,  6,  6,  6,  4,  5,  6,
-    4,  6,  6,  6,  6,  6,  5,  6,  6,  6,  6,  6,  6,  6,  6,  8,
+    6,  6,  6,  6,  6,  6,  6,  6,  6,  6,  3,  4,  6,  6,  3,  6,
+    6,  6,  5,  6,  6,  6,  6,  6,  6,  6,  6,  6,  6,  3,  6,  8,
     3,  6,  6,  6,  6,  6,  6,  3,  3,  3,  3,  3,  3,  3,  3,  3,
     10, 10, 10, 10,  8, 10, 10,  8, 10, 10, 10, 10, 10, 10, 10, 10,
     10, 10, 10, 10, 10, 10, 10, 10,  3,  3,  3,  3,  3,  3,  3,  3,
